README.md

```diff
@@ -95,11 +95,11 @@ By default, if `data-i18n-attr` is not defined, the `innerHTML` will be translat
 
 ### Translating programmatically
 
-Alternatively, you can translate a single, given key via the method `getTranslationByKey(lang, key)`. The first argument should be a valid language string like "en" or "de", the second argument should be a key from your translation files, such as "header.title".
```

Comment:
👍

Reply:
damn, that's wrong... We handle the fallback at `_getValueFromJSON` and not at `getTranslationByKey`.
Sorry!
Reverted in https://github.com/andreasremdt/simple-translator/pull/8